src/main/java/com/cloudant/client/api/views/ViewRequest.java

```diff
@@ -46,6 +46,37 @@
     ViewResponse<K, V> getResponse() throws IOException;
 
     /**
+     * Performs the request for the given parameter ID and returns the response.
```

Comment:
I'm not sure parameter is the best name here, as both parameter and ID terms have a bunch of meanings here (e.g., I thought you meant query string parameter for a bit).

I'd go for something pagination specific, like `paginationToken`.